File: samples/python-v2-azurefunctions/retrieve_secret.py
import json
import logging 
import azure.functions as func 

retrieveSecret = func.DaprBlueprint()

# Dapr secret input binding with http dapr_service_invocation_trigger
@retrieveSecret.function_name(name="RetrieveSecret")
@retrieveSecret.dapr_service_invocation_trigger(arg_name="payload", method_name="RetrieveSecret")
@retrieveSecret.dapr_secret_input(arg_name="secret", secret_store_name="localsecretstore", key="my-secret", metadata="metadata.namespace=default")
def main(payload, secret: str) :
    # Function should be invoked with this command: dapr invoke --app-id functionapp --method RetrieveSecret  --data '{}'
    logging.info('Python function processed a RetrieveSecret request from the Dapr Runtime.')
    secret_dict = json.loads(secret)

    for key in secret_dict:
        logging.info("Stored secret: Key = " + key +
                     ', Value = ' + secret_dict[key])

File: samples/javascript-v4-azurefunction/src/functions/retrieveSecret.js
const { app, input, trigger } = require('@azure/functions');

const daprSecretInput = input.generic({
    type: "daprSecret",
    secretStoreName: "localsecretstore",
    metadata: "metadata.namespace=default",
    direction: "in",
    name: "secret",
    key: "my-secret"
});

app.generic('RetrieveSecret', {
    trigger: trigger.generic({
        type: 'daprServiceInvocationTrigger',
        name: "payload"
    }),
    extraInputs: [daprSecretInput],
    handler: async (request, context) => {
        context.log("Node function processed a RetrieveSecret request from the Dapr Runtime.");
        const daprSecretInputValue = context.extraInputs.get(daprSecretInput);

        // print the fetched secret value
        for (var key in daprSecretInputValue) {
            context.log(`Stored secret: Key=${key}, Value=${daprSecretInputValue[key]}`);
        }
    }
});